test/ffmpeg-vaapi/encode/encoder.py

```diff
@@ -212,10 +216,15 @@ def check_output(self):
 
     # ipb mode
     ipbmode = 0 if vars(self).get("gop", 0) <= 1 else 1 if vars(self).get("bframes", 0) < 1 else 2
+    if vars(self).get("lowdelayb", None) is not None:
+      assert type(self.lowdelayb) is int and self.lowdelayb >= 0 and self.lowdelayb <= 2, "invalid lowdelayb value"
+      if 1 == self.lowdelayb :
+        ipbmode = 3
     ipbmsgs = [
       "Using intra frames only",
       "Using intra and P-frames",
-      "Using intra, P- and B-frames"
+      "Using intra, P- and B-frames",
+      "delay" #cqp output is "Low delay", cbr/vbr output is "low delay"
```

Comment:
a little confuse about the comment, are you saying "cqp" and "cbr/vbr" have the same output ?

Reply:
pay attention to that there is caps for "Low" and "low" for ffmpeg output log, so this patch only search  the second word mach

Reply:
Since these are passed as regular expressions to `re.match`, then maybe you could use `"[L|l]ow delay"` instead (you may need to check this syntax, though)?

Reply:
Ok, I am trying

Reply:
yes, updated based on your suggestion